runtime/onert/frontend/nnapi/wrapper/ANeuralNetworksCompilation.cc

```diff
@@ -26,9 +26,7 @@ ANeuralNetworksCompilation::ANeuralNetworksCompilation(const ANeuralNetworksMode
     _compiler{std::make_shared<compiler::Compiler>(_model, *_coptions)}
 {
   if (model->allowedToFp16())
-  {
-    _compiler->enableToFp16();
-  }
+    _coptions->enableToFp16();
```

Comment:
(Optional) Can one line code be used without a block? I thought it would be changed in code format. It's interesting. :)